www/waterfall.inc

```diff
@@ -1958,8 +1956,7 @@ function GetAverageSliceValue($slices, $start, $end)
             $start_value = floatval($slices[$start_key]);
             $end_value = floatval($slices[$end_key]);
             $end_weight = floatval($start - $start_bucket) / 0.1;
-            $start_weight = 1.0 - $distance_from_start;
-            $avg = $start_weight * $start_value + $end_weight * $end_value;
```

Comment:
The math here looks wrong.  It looks like $distance_from_start might not have been set and I can't remember exactly what it was supposed to be doing but it looks like it was supposed to be using part of each of the start and end values and removing the start_weight entirely doesn't fix the math (though probably makes it do what the code was actually doing - but not what it was intended to do).

Reply:
yeah, `$distance_from_start` is never defined anywhere and so in PHP 1.0 - undefined var is 1, so `$start_value` times 1 is always `$start_value`. But I see what you mean about the logic. I'll just define `$distance_from_start = 0` and leave it alone